salesforce/10.14/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -1213,18 +1243,19 @@ Converts a Lead into an Account, Contact, or (optionally) an Opportunity.
 
 Adds one or more new records to your organization's data.
 
-IMPORTANT: When you map your objects to the input of this message processor, keep in mind that the objects need to match the expected type of the object at Salesforce. For example, if you set the CloseDate field of an Opportunity to a string of value "2011-12-13", it is sent to Salesforce as a string. The operation will be rejected because the CloseDate is not of the expected type. The correct way to map it is to generate a Java Date object. You can do so using the Groovy expression evaluator as `#[groovy:Date.parse("yyyy-MM-dd", "2011-12-13")]`.
+[IMPORTANT]
+When you map your objects to the input of this message processor, the objects must match the expected type of the object at Salesforce. For example, if you set the CloseDate field of an opportunity to a string of value `2011-12-13`, it is sent to Salesforce as a string. The operation is rejected because the CloseDate is not of the expected type. The correct way to map it is to generate a Java Date object. You can do so using the Groovy expression evaluator as `#[groovy:Date.parse("yyyy-MM-dd", "2011-12-13")]`.
```

Comment:
```suggestion
When you map your objects to the input of this message processor, the objects must match the expected type of the object at Salesforce. For example, if you set the *CloseDate* field of an opportunity to a string of value `2011-12-13`, it is sent to Salesforce as a string. The operation is rejected because the CloseDate is not of the expected type. The correct way to map it is to generate a Java Date object. You can do so using the Groovy expression evaluator as `#[groovy:Date.parse("yyyy-MM-dd", "2011-12-13")]`.
```